tests/ebpf/ebpf_thread_function.sh.in

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+netdata_ebpf_test_functions() {
+    curl -k -o /tmp/ebpf_netdata_test_functions.txt "${1}"
+    TEST=$?
+    if [ $TEST -ne 0 ]; then
+        if [ -f /tmp/ebpf_netdata_test_functions.txt ]; then
+            rm /tmp/ebpf_netdata_test_functions.txt
+        fi
+        echo "Cannot request run a for ${1}."
+        exit 1
+    fi
+
+    grep "${2}" /tmp/ebpf_netdata_test_functions.txt >/dev/null
+    if [ $TEST -ne 0 ]; then
```

Comment:
Hi Thiago. Is there a `TEST=$?` missing right above this line here?

Reply:
Last commit fixed this, thanks!